packages/helper-grammar-regex-collection/test.js

```diff
@@ -89,6 +116,18 @@ const fixtures = {
       'require.resolve (foo)',
       'require.resolve("fo o")',
       'requireDresolve("foo")',
+      // JavaScript Dynamic Imports import("./module")
+      'import(foo)',
+      'import"foo"',
+      'import ("foo")',
+      'import (foo)',
+      'import("fo o")',
+      // import.resolve
+      'import.resolve(foo)',
+      'import.resolve"foo"',
+      'import.resolve ("foo")',
+      'import.resolve (foo)',
+      'import.resolve("fo o")'
```

Comment:
The CI is failing, because there's a linting error see https://travis-ci.org/OctoLinker/browser-extension/builds/213108188#L465 Adding a trailing comma will fix it